script/smoke_test.sh

```diff
@@ -4,7 +4,7 @@
 
 SCALA_SCALA_LIBRARY_EXPECTED=87
 SCALA_SCALA_COMPILER_EXPECTED=45
-DOTTY_COMPILER_EXPECTED=56
+DOTTY_COMPILER_EXPECTED=55
```

Comment:
Note 1% regression on Dotty compiler.

Reply:
Odd, I'd expect this to bump the coverage up, not down. Any idea why?

Reply:
With these small percentage difference, it's hard to tell. Potentially some files were parsed incorrectly before, and now that givens is interpreted correctly it's hitting some other unsupported feature?